### DISCUSSION TITLE
Oracle Linux KVM Support for CloudStack

### DISCUSSION THREAD

**Discussion by farhad-aslam** [apache/cloudstack#8572] [Q&A] [answered]
Is oracle Linux based KVM supported as a hypervisor with CloudStack? I cannot see it anywhere in the documentation.

**Comment by DaanHoogland** [comment, marked as answer]
ol8 and ol9 are supported as regular el varyities. Older vversions were supported by specific OL plugins but that was less stable and mature.

**Reply by yadvr** [reply]
Both Oracle Linux 8 and 9 are RHEL clones and therefore supported as management server and KVM hosts with Apache CloudStack. We regularly run smoketests on pull requests against ol8/ol9. We should update the compat matrix with the 4.19 release and onwards.

**Comment by yadvr** [comment]
The latest 4.19 release notes compatibility matrix fixed that. @farhad-aslam closing on this remark.
